ember/app/components/form.hbs

```diff
@@ -8,7 +8,10 @@
         model=@model
         loading=this.loading
         input=(component
-          "validated-input" model=@model submitted=this.submitted
+          "validated-input"
+          model=@model
+          submitted=this.submitted
+          validateModel=this.validateModel
```

Comment:
I don't see `this.validateModel` being defined anywhere.

Reply:
`this.validateModel` was lost when rebasing after #170 was merged (exact commit: https://github.com/adfinis/Outdated/pull/170/commits/51940586953e21bc04cbb6f021a7813a4cf13213)

I have now re implemented this:
https://github.com/adfinis/Outdated/blob/963398af242e6dccb65d97490ccd189a720b7ee9/ember/app/components/form.js#L10-L21
https://github.com/adfinis/Outdated/blob/963398af242e6dccb65d97490ccd189a720b7ee9/ember/app/components/validated-input/render/component.js#L8-L14